borg/archiver.py

```diff
@@ -652,9 +648,10 @@ def preprocess_args(self, args):
 
     def build_parser(self, args=None, prog=None):
         common_parser = argparse.ArgumentParser(add_help=False, prog=prog)
-        common_parser.add_argument('-v', '--verbose', dest='verbose', action='store_true', default=False,
-                                   help='verbose output')
-        common_parser.add_argument('--log-level', dest='log_level', default='info', metavar='LEVEL',
+        common_parser.add_argument('-v', '--verbose', dest='log_level',
+                                   action='store_const', const='info', default='warning',
+                                   help='verbose output, same as --log-level=info')
+        common_parser.add_argument('--log-level', dest='log_level', default='warning', metavar='LEVEL',
```

Comment:
> Hmm, is this a problem if 2 arguments work on the same "dest" variable?

Yes, it is a problem. I *think* the last one wins, but i was considering bailing out if two options of the kind were specified.

> I mean might we get strange difference depending on the order people use these?

Yes, we might.

> Suggestion for -v: help="switch to log level to info (if higher than info)"
> Of course it must not switch to info, if --log-level=debug was given.

Right. One thing i was considering was to just add `--verbose` and `--debug` as aliases to the `--log-level` option and make a special action handler for those.

> Switching back to warning default might be a good idea to shut it up (quiet by default).

I believe i did that already...

> Just needs some checking whether stuff that should get through gets through (without emitting at insane log level).

... and all tests pass (although `check` tests needed to be patched).

> btw, other flags that enable info-level output could also trigger adjusting the level (or you won't see the output).
> 
> Hmm, somehow this still feels strange (but we're getting closer maybe).

hopefully.